draft-duke-httpbis-quic-version-alt-svc.md

```diff
@@ -132,6 +142,12 @@ If the Alt-Svc information resolves to a server pool that inconsistently
 supports different QUIC versions, the parameter SHOULD only advertise versions
 that are supported throughout the pool.
 
+# The quicv SvcParamKey
+
+SVCB and HTTPS Resource Records can include the quicv SvcParamKey. Its presentation
+format value and use are identical to the quicv Alt-Svc Parameter.  Its wire format value
+consists of the version numbers in network byte order.
```

Comment:
Personally, I think it would help to see an illustrative example of one of these. That would help compare Alt-Svc to the RR when `quicv` is in play, similar https://datatracker.ietf.org/doc/html/draft-ietf-dnsop-svcb-https-08#section-9.3